test/test_transforms.py

```diff
@@ -1317,6 +1317,23 @@ def test_random_grayscale(self):
         # Checking if RandomGrayscale can be printed as string
         trans3.__repr__()
 
+    def test_random_erasing(self):
```

Comment:
Can you make the test a bit more stronger by checking if the region around the erased patch is equal to the original image?

Reply:
Addressed by #1060.